### PR TITLE
EventsTabsActivity: use actual data and listen to changes

### PR DESCRIPTION
**Pull request by AmitSilber**

### CODE REVIEW

**Review by mfocshaner — DISMISSED** [review]
i hope it works

**Comment by idosavion** [comment]
Please add current event and user using putExtra(), scrolling up for updating looks good

**Comment by AmitSilber** [comment]
> Please add current event and user using putExtra(), scrolling up for updating looks good

ok. Checked and you are not using usermodel so I removed it and accessed usermodelid via firebase

File: app/src/main/java/com/huji/foodtricks/buddies/EventsTabsActivity.java [hunk 1]
```diff
@@ -1,49 +1,150 @@
...
+
+
+    final DatabaseStreamer streamer = new DatabaseStreamer();
+    DatabaseReference DBref;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        //this.spinner =  findViewById(R.id.progressBar1);
+        //this.spinner.setVisibility(View.GONE);
```
Comment (mfocshaner) [inline app/src/main/java/com/huji/foodtricks/buddies/EventsTabsActivity.java:60]: remember to remove if we decide not to use it

File: app/src/main/java/com/huji/foodtricks/buddies/EventsTabsActivity.java [hunk 2]
```diff
@@ -1,49 +1,150 @@
...
+                for (final String eventId : newEvents) {
+                    streamer.fetchEventModelById(eventId, new EventFetchingCompletion() {
+                        @Override
+                        public void onFetchSuccess(EventModel model) {
+                            sendNewEventSwitch(eventId, model);
+                        }
+
+                        @Override
+                        public void onNoEventFound() {
+                            // cry :(
```
Comment (mfocshaner) [inline app/src/main/java/com/huji/foodtricks/buddies/EventsTabsActivity.java:95]: lol

File: app/src/main/java/com/huji/foodtricks/buddies/Models/UserModel.java [hunk 3]
```diff
@@ -106,4 +107,28 @@ public boolean groupNameTaken(String groupName) {
...
+    public void addChangedEvent(String changes) {
+        if (this.changedEvents == null) {
+            this.changedEvents = new ArrayList<>();
+        }
+        this.changedEvents.add(changes);
+    }
+
+    public void clearChangedEvents() {
+        if (this.changedEvents == null) {
+            this.changedEvents = new ArrayList<>();
```
Comment (mfocshaner) [inline app/src/main/java/com/huji/foodtricks/buddies/Models/UserModel.java:129]: tahlas, can just return

File: app/src/main/res/layout/fragment_events_tabs.xml [hunk 4]
```diff
@@ -30,15 +33,23 @@
         android:layout_height="wrap_content"
         android:layout_gravity="bottom|end"
         android:layout_margin="@dimen/fab_margin"
+        android:clickable="true"
         android:src="@drawable/plus"
-        app:backgroundTint="@color/colorAccent"
-        android:clickable="true"/>
+        app:backgroundTint="@color/colorAccent" />
+
+    <!--ProgressBar
```
Comment (mfocshaner) [inline app/src/main/res/layout/fragment_events_tabs.xml:40]: can remove

**Review by mfocshaner — APPROVED** [review]
good job!

**Pull request merged**